notebooks/ImvizExample.ipynb

```diff
@@ -596,35 +596,65 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "d6af9b27-e5b6-45b6-ac85-5d008faa88b1",
+   "id": "0796b7c5",
    "metadata": {},
    "outputs": [],
    "source": [
-    "imviz.save('myimage.png')"
+    "viewer.save('myimage.png')"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "96fee8ea-66e3-489c-be27-b06223dd8809",
+   "id": "5e64af2b",
    "metadata": {},
    "source": [
-    "If you want to align the images by WCS instead of pixels, you can run this but it will remove all markers."
+    "If you want to align the images by WCS instead of pixels, you can run this but it will remove all markers added by the default viewer.\n",
+    "\n",
+    "If you have added markers using a tiled viewer you manually created yourself (see below), you have to call the `reset_markers()` method for that viewer first before running the next cell."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "0e59c379-20ec-4125-83eb-07f56ac22f9b",
+   "id": "b4f68d86",
    "metadata": {},
    "outputs": [],
    "source": [
     "imviz.link_data(link_type='wcs')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "7a4f7c3f",
+   "metadata": {},
+   "source": [
+    "If you want a second viewer, click on the \"picture with +\" icon on top. You can also programmatically control the new viewer, as follows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77368ba8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viewer_2 = imviz.app.get_viewer_by_id('imviz-1')"
```

Comment:
I found this really confusing.  The initial viewer was imviz-0, which confused me that near the top the `viewer = imviz.app.get_viewer_by_id('imviz-1')` worked.  But then even more confusingly, this *also* worked, even though it seems like it should have gotten the *first* viewer.

We should probably clean this up code-wise somehow, but if that's going to be a lot of work, for now, we can remediate it by just adding a sentence or two explaining it in the text above the cells?

Reply:
I don't even know how to begin with this one... The internals are confusing even for me. There is a viewer id and then there is the viewer name and somehow they are both tracked differently and one of them is optional. This gives me bad memory from VOTables.

Reply:
@eteq and I had agreed to the following resolutions:

* Change the YAML to have the default viewer be named `imviz-0` to be consistent with its ID.
* Have a new property `imviz.default_viewer` that would return its instance.